### PR TITLE
Add check_server_cert.sh

### PR DESCRIPTION
**Pull request by aaronpkahn**
# Description

Adds a script `tools/check_server_cert.sh` to validate client configs against telemetry servers.

## Type of change

Please select all options that apply to this change:

- [X] New feature (non-breaking change which adds functionality)
- [X] Documentation update

# Checklist:
- [X] My code follows the style of this project.
- [X] I have performed a self-review of my code.
- [X] I have made corresponding updates to the documentation.

### CODE REVIEW

**Review by Castaglia — APPROVED** [review]

**Review by agbpatro — APPROVED** [review]

File: tools/check_server_cert.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    exit 1
+}
+
+CONFIG=$1
+
+if [ ! -f "$CONFIG" ]; then
+  die "could not find json config file: $CONFIG"
+fi
+
+HOSTNAME=$(jq -r ".hostname" "$CONFIG")
```
Comment (SirArkimedes) [inline tools/check_server_cert.sh:31]: Should this be `.host`?
https://github.com/teslamotors/fleet-telemetry/blob/613041e96d20ceba01a731fe2d42d4e0b5c6440e/config/config.go#L31
```suggestion
HOSTNAME=$(jq -r ".host" "$CONFIG")
```

File: tools/check_server_cert.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+CA=$(jq -r ".ca" "$CONFIG")
```
Comment (SirArkimedes) [inline tools/check_server_cert.sh:32]: Should this be `.ca_file`?
https://github.com/teslamotors/fleet-telemetry/blob/613041e96d20ceba01a731fe2d42d4e0b5c6440e/config/config.go#L131

```suggestion
CA=$(jq -r ".ca_file" "$CONFIG")
```

Comment (SirArkimedes) [reply tools/check_server_cert.sh:32]: Ohhhh. Okay, the new README updates make this make sense, then! Thanks!

File: tools/check_server_cert.sh (hunk 1 above)
Comment (aaronpkahn) [reply tools/check_server_cert.sh:31]: @SirArkimedes thanks for the feedback, but this script is for validating server setup is compatible with a proposed vehicle config (which uses "hostname"). I've updated the README, and error messaging.

**Pull request merged**